client/components/app.jsx

```diff
@@ -1,8 +1,41 @@
 import React from 'react';
+import PageTitle from './page-title';
+import GradesTable from './grades-table';
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      grades: []
+    };
+    this.getAllGrades = this.getAllGrades.bind(this);
+  }
+
+  componentDidMount() {
+    this.getAllGrades();
+  }
+
+  getAllGrades() {
+    fetch('/api/grades')
+      .then(res => res.json())
+      .then(grade => {
+        this.setState({
+          grades: grade
+        });
+      });
+  }
+
   render() {
-    return null;
+    return (
+      <div className="container col-sm-12 col-lg-8">
```

Comment:
Bootstrap grid classes are typically set up as
```html
<div class="container">
  <div class="row">
    <div class="col-screensize-width"></div>
    <div class="col-screensize-width"></div>
    <div class="col-screensize-width"></div>
    <div class="col-screensize-width"></div>
    etc etc
  </div>
</div>
```

I recommend avoiding mixing container, row, or col classes as it can lead to some pretty unexpected behavior. The extra divs might be annoying, but the clashing styles are much less confusing.

Reply:
> but the clashing styles are much less confusing.
i mean much _more_ confusing